ember/app/components/cases-table/component.js

```diff
@@ -41,18 +35,23 @@ export default class CasesTableComponent extends Component {
 
   @restartableTask
   *fetchCases(cursor = null) {
+    const order = (this.args.order || ENV.APP.casesTable.defaultOrder).split(
```

Comment:
Are those `description` properties still useful, which are defined in the `environment.js` file?

Reply:
@derrabauke the code still depend on the defined environment variables